common/configdir.c

```diff
@@ -22,11 +22,21 @@ static char *default_configdir(const tal_t *ctx)
 	return path;
 }
 
+static char *default_rpc_file(const tal_t *ctx)
+{
+	char *path;
+	
+    path = path_basename(ctx, "lightning-rpc");
+
+	return path;
```

Comment:
Inconsistent indentation, please use tabs (https://www.youtube.com/watch?v=SsoOG6ZeyUI) :wink:

Reply:
^^ removed some whitespace valgrind lammers..

The issue with this rpc-file came up when i want to call the lightning-cli  with options from PHP 
this still does not work.
Do i have to open the unix socket directly in a PHP script?